SPECS/forkexecd.spec

```diff
@@ -1,12 +1,12 @@
-%global package_speccommit cbd05b8df11622176d97ff78aace63d7523bd01b
-%global package_srccommit v1.18.1
-Version: 1.18.1
+%global package_speccommit 3a70be6b5605eae565960c1ea1a6ef9f0474fbd1
+%global package_srccommit v1.18.3
+Version: 1.18.3
 Release: 2%{?xsrel}%{?dist}
```

Comment:
Given `Version` is bumped up, do we want to keep `Release` at 2, should it not go back to 1?

Reply:
In this one here what from upstream:
```
Version: 1.18.3
Release: 2%{?xsrel}%{?dist}
```
So i keep the `2`, but add a `.1` after for our version